packages/docusaurus-theme-classic/src/theme/CodeBlock/index.js

```diff
@@ -117,6 +119,7 @@ export default ({children, className: languageClassName, metastring}) => {
               {showCopied ? 'Copied' : 'Copy'}
             </button>
             <pre
+              tabIndex="0"
```

Comment:
We could add this attribute to the `div` element with `codeBlockLines` class, but then the outline is poorly visible, so I added it to the `pre` tag and disabled the corresponding ESLint rule.